FirebaseRemoteConfig/Tests/Unit/RCNConfigContentTest.m

```diff
@@ -17,12 +17,12 @@
 #import <XCTest/XCTest.h>
 
 #import <OCMock/OCMock.h>
-#import "FIRRemoteConfig.h"
-#import "RCNConfigContent.h"
-#import "RCNConfigDBManager.h"
-#import "RCNConfigSettings.h"
-#import "RCNConfigValue_Internal.h"
-#import "RCNTestUtilities.h"
+#import "FirebaseRemoteConfig/Sources/Public/FIRRemoteConfig.h"
```

Comment:
Should I add a linter check for this?

Reply:
Sure, would be nice to catch before copybara does